app/controllers/issues_controller.rb

```diff
@@ -95,6 +100,10 @@ def load_valid_assignees
     @valid_assignees = @project.valid_assignees
   end
 
+  def load_issue_watcher
```

Comment:
implement 'Show all watchers' on issues detail page for admins.

Reply:
implemented

Reply:
done